todo_app/data/test_view_model.py

```diff
@@ -0,0 +1,30 @@
+from todo_app.data.ViewModel import ViewModel
+from todo_app.data.Item import Item
+
+not_started_item_1 = Item(1, "to do 1", "Not started")
+not_started_item_2 = Item(1, "to do 1", "Not started")
+done_item = Item(2, "to do 2", "Done")
+test_items = [not_started_item_1, done_item, not_started_item_2]
+
+
+def test_view_model_correctly_filters_to_items_with_not_started_status():
+    # Arrange
+    view_model = ViewModel(test_items)
```

Comment:
Not a strong recommendation given that you're _probably_ assuming this data at the top shouldn't be changed during any tests, but worth noting that if someone did (intentionally or accidentally) start modifying this then we might end up with tests that depend on the order in which they're run

One alternative might be to generate this data in a fixture, so that each test can get a fresh copy:
```suggestion
import pytest

@pytest.fixture
def test_items():
    not_started_item_1 = Item(1, "to do 1", "Not started")
    not_started_item_2 = Item(1, "to do 1", "Not started")
    done_item = Item(2, "to do 2", "Done")
    return [not_started_item_1, done_item, not_started_item_2]

def test_view_model_correctly_filters_to_items_with_not_started_status(test_items):
    # Arrange
    view_model = ViewModel(test_items)
```